crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -84,6 +85,17 @@ impl CommandBase {
         let global_config_path = config_dir.join("turborepo").join("config.json");
         AbsoluteSystemPathBuf::try_from(global_config_path).map_err(ConfigError::PathError)
     }
+    /// Returns the path to the global auth file (auth.json).
+    fn global_auth_path(&self) -> Result<AbsoluteSystemPathBuf, ConfigError> {
+        #[cfg(test)]
+        if let Some(global_auth_path) = &self.global_auth_path {
+            return Ok(global_auth_path.clone());
+        }
```

Comment:
I've seen in the rest of the codebase that the `cfg(test)` attribute is typically on entire functions, rather than blocks inside a function. Should we continue to follow that pattern?

Reply:
There's some other spots where we do this, like in the `TurborepoConfigBuilder`